hostscripts/clouddata/syncSLErepos

```diff
@@ -326,6 +326,11 @@ if [ $SITE = "nue" ]; then
 
     # this one is always failing - we need to ignore that otherwise the lock file remains stuck
     timeout 1h ~/bin/syncgitrepos || :
+
+else
+    # Keep stuff that is "owned" in NUE site over to this site
+    $rsync --exclude='*untested*'  clouddata.cloud.suse.de::cloud/images/x86_64/ /srv/nfs/images/x86_64/
```

Comment:
This reads like it would overwrite something that was synced above, search for `destdir_admin_images=/srv/nfs/images/`. Same on the next line but search for `/srv/nfs/repos/`.

Reply:
@JanZerebecki good finding on the admin images, I fixed that in a followup commit since its strictly not a regression here I think. I don't quite get the point as disabled

Reply:
Missed that no slash...